packages/api/package.json

```diff
@@ -4,11 +4,15 @@
   "main": "./src/index.ts",
   "types": "./src/index.ts",
   "license": "MIT",
+  "scripts": {
+    "typecheck": "tsc --noEmit"
+  },
   "dependencies": {
+    "@agreeto/auth": "workspace:*",
     "@agreeto/db": "workspace:*",
-    "@trpc/client": "10.0.0-proxy-beta.17",
-    "@trpc/server": "10.0.0-proxy-beta.17",
-    "superjson": "^1.9.1",
+    "@trpc/client": "10.0.0-rc.1",
+    "@trpc/server": "10.0.0-rc.1",
+    "superjson": "1.9.1",
```

Comment:
superjson 1.10 is unreliable on certain runtimes and doesn't provide any extra feature so locking it at 1.9

Reply:
TIL. Good to know.